src/Engine/Combat/CombatEngine.java

```diff
@@ -11,13 +11,9 @@
 
   public static void resolveCombat(Unit attacker, Unit defender, GameMap map)
   {
-    // TODO: make sure to clean up unneeded modifiers
-    //		for(int i = 0; i < modifiers.size(); i++) {
-    //			if (modifiers.get(i).done) {
-    //				modifiers.get(i).initTurn();
-    //				modifiers.remove(i);
-    //			}
-    //		}
+    // The list of modifiers is updated and culled every turn in GameInstance.
+    // It is emptied in MapController when exiting the map.
+    // TODO: Figure out a better way to handle their lifecycle?
```

Comment:
Would it make more sense to have Commander own their CombatModifiers? This would clean up the lifecycle quite a bit I think. The Commander's initTurn could be used to remove any that no longer matter, and then we don't have to remember to clean out the engine at the end of the game. Plus, I'm just not sure I like having a global, static entity that persists between games own data for a specific game.